server/lib/platform.js

```diff
@@ -0,0 +1,17 @@
+import { platform } from 'os';
```

Comment:
You don't need that, we already have `electron-platform` (https://www.npmjs.com/package/electron-platform). And it has all the platform detection functions you need. I think this also gets rid of the `os` dependency.